UnitsNet.Tests/CustomCode/FrequencyTests.cs

```diff
@@ -24,8 +24,10 @@ public class FrequencyTests : FrequencyTestsBase
 
         protected override double CyclesPerMinuteInOneHertz => 60;
 
-        protected override double RadiansPerSecondInOneHertz => 2*Math.PI;
+        protected override double RadiansPerSecondInOneHertz => 2 * Math.PI;
 
         protected override double BeatsPerMinuteInOneHertz => 60;
+
+        protected override double BUnitsInOneHertz => Math.Sqrt(1000);
```

Comment:
No calculations here please, prefer constants. See guidelines:

https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit#4-fix-generated-test-stubs-to-resolve-compile-errors-tests

Reply:
Done.